lib/open_project/costs/engine.rb

```diff
@@ -351,6 +351,15 @@ class Engine < ::Rails::Engine
                                                attribute: :updated_on
     end
 
+    module EagerLoadedCosts
+      def eager_loaded_work_packages(ids)
```

Comment:
FYI I took the liberty of pushing the [minor refactoring](https://github.com/opf/openproject/commit/c9272a11c98f1e624d199a5259f9994a68c9042f) allowing this to the core.